stream_alert_cli/test.py

```diff
@@ -146,21 +150,22 @@ def test_kinesis_alert_rules():
             with open(rule_file_path, 'r') as rule_file_handle:
                 try:
                     contents = json.load(rule_file_handle)
-                except ValueError:
-                    LOGGER_CLI.info('Error loading %s, bad JSON', rule_file)
+                except ValueError as err:
+                    LOGGER_CLI.error('Error loading %s: %s', rule_file, err)
+                    continue
 
-            test_records = contents['records']
-            if len(test_records) == 0:
-                LOGGER_CLI.info('No records to test for %s', rule_name)
+                test_records = contents['records']
```

Comment:
The except block captures if the file was opened successfully, but the JSON fails to parse.

However, if the file fails to open at all in `with open(..`, having this outside of the `with` block will still cause issues.. yes?